programs/driver_board/main.c

```diff
@@ -19,7 +25,11 @@ void execute_master_command(uint16_t command_id, uint8_t *buff) {
 
         case 0xBB:
             // Motor movement command
-
+            if(buff[0] == 0) {
+                motor_speed = -1 * (buff[1] * 1000 + buff[2] * 100 + buff[3] * 10 + buff[4]);
+            } else {
+                motor_speed = (buff[1] * 1000 + buff[2] * 100 + buff[3] * 10 + buff[4]);
+            }
```

Comment:
Please change this to:
```c
motor_speed = buff[1] * 1000 + buff[2] * 100 + buff[3] * 10 + buff[4];
if (buff[0] == 0) motor_speed *= -1;
```

Also can I just ask why `0` as negative value marker? Usually `1` is used (as in, for example, two's complement for binary numbers, where sign bit is `0` for positive values and `1` for negative ones). I'm just wondering why this choice?